src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -26,6 +26,8 @@
 
     <!-- By default we'll build libraries referenced by packages -->
     <BuildPackageLibraryReferences Condition="'$(BuildPackageLibraryReferences)' == ''">true</BuildPackageLibraryReferences>
+
+    <BuildInParallel Condition="'$(BuildInParallel)' == '' AND '$(MSBuildNodeCount)' > '1'">true</BuildInParallel>
```

Comment:
Yes, BuildInParallel isn't set by default and MSBuild doesn't seem to provide any other indication of wether it's been run in parallel other than this Node count.  So this is really just setting BuildInParallel to whatever the user specified on the command line (/m => true)